src/libServer/LookupServer.cpp

```diff
@@ -1183,6 +1183,23 @@ Json::Value LookupServer::GetEthBlockCommon(const TxBlock& txBlock,
                                                  includeFullTransactions);
 }
 
+Json::Value LookupServer::GetEthBalance(const std::string& address) {
+  const auto balanceStr = this->GetBalance(address, true)["balance"].asString();
+
+  const uint256_t ethBalance =
+      std::strtoll(balanceStr.c_str(), nullptr, 16) * EVM_ZIL_SCALING_FACTOR;
+
+  std::stringstream strm;
+  strm << "0x" << std::hex << ethBalance;
+
+#if 1
```

Comment:
Is it a leftover from testing? If we need these debug calls, can we remove conditional `if 1` otherwise?